apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx

```diff
@@ -1,9 +1,149 @@
+                        {phones[0] ?? noData}
+                    </span>
+                </div>
+
+                <div className={styles.listItemData}>
+                    <label htmlFor="otherNames" className={styles.listItemLabel}>
+                        OTHER NAMES
+                    </label>
+                    <br />
+                    <div id="otherNames">
+                        {names.length ? (
+                            names?.map((name, index) => (
+                                <div key={index}>
+                                    <span className={styles.value}>
+                                        {name.last}, {name.first}
+                                    </span>
+                                    <br />
+                                </div>
+                            ))
+                        ) : (
+                            <span className={styles.value}> {noData}</span>
+                        )}
+                    </div>
+                </div>
+            </div>
+            <div className={styles.listItemBox}>
+                <div className={styles.listItemData}>
+                    <label htmlFor="email" className={styles.listItemLabel}>
+                        Email
+                    </label>
+                    <br />
+                    <span id="email" className={styles.value}>
+                        {emails[0] ?? noData}
+                    </span>
+                </div>
+
+                <div className={styles.listItemData}>
+                    <label htmlFor="address" className={styles.listItemLabel}>
+                        address
+                    </label>
+                    <br />
+                    {addresses.length ? (
+                        <>
+                            <span id="email" className={styles.value}>
+                                {addresses[0].address}
+                            </span>
+                            <br />
+                            <span>{`${addresses[0].city}, ${addresses[0].state}, ${addresses[0].zipcode}`}</span>
+                        </>
+                    ) : (
+                        <span className={styles.value}>{noData}</span>
+                    )}
+                </div>
+            </div>
+            <div className={styles.listItemBox}>
+                <div className={styles.listItemData}>
+                    <label htmlFor="driversLicense" className={styles.listItemLabel}>
+                        driver's license
+                    </label>
+                    <br />
+
+                    <span id="driversLicense" className={styles.value}>
+                        {driversLicense?.value ?? noData}
+                    </span>
+                </div>
+
+                <div className={styles.listItemData}>
+                    <label htmlFor="socialSecurityNumber" className={styles.listItemLabel}>
+                        Social Security
```

Comment:
question: should we also consider other id types? or are we only keeping these always?

Reply:
The designs only show these two identification types, will have to consult with the design team

Reply:
this is what was displayed on the figma so i went with it